src/controllers/PostController.ts

```diff
@@ -1,32 +1,33 @@
-import {JsonController, Get, Post as HttpPost, Param, Delete, Body} from "routing-controllers";
+import {JsonController, Get, Post, Param, Delete, Body} from "routing-controllers";
 import {Service} from "typedi";
 import {PostRepository} from "../repository/PostRepository";
-import {Post} from "../model/Post";
+import {Post as PostModel} from "../model/Post";
```

Comment:
I haven't understand at the first glance when I looked at `@HttpPost("/posts")` about what it does, I have to checked imports like hmm, okay so where is this coming from? While using `PostModel` makes it immediately clear that it's a model. I think people will recognize `PostModel` faster than `@HttpPost`. If we want to keep things simple here, we should keep this way I think.